docs/docs/api/expansion/api-reference.md

```diff
@@ -16,34 +16,134 @@ Since this service wraps other services, the request format mostly follows the o
 |:----------------------------------| :------------------------------------ |
 | `action` (required)               | The service whose expansion should be tracked. Currently one of `route`, `isochrone` or `sources_to_targets`. | 
 | `skip_opposites` (optional)       | If set to `true` the output won't contain an edge's opposing edge. Opposing edges can be thought of as both directions of one road segment. Of the two, we discard the directional edge with higher cost and keep the one with less cost. Default false. | 
-| `expansion_properties` (optional) | A JSON array of strings of the GeoJSON property keys you'd like to have in the response. One or multiple of "durations", "distances", "costs", "edge_ids", "statuses". **Note**, that each additional property will increase the output size by minimum ~ 25%. By default an empty `properties` object is returned. |
+| `expansion_properties` (optional) | A JSON array of strings of the GeoJSON property keys you'd like to have in the response. One or multiple of "duration", "distance", "cost", "edge_id", "pred_edge_id" or "edge_status". **Note**, that each additional property will increase the output size by minimum ~ 10%. By default an empty `properties` object is returned. |
 
 The `expansion_properties` choices are as follows:
 
 | Property   | Description                           |
 | :--------- | :------------------------------------ |
-| `distances`   | Returns the accumulated distance in meters for each edge in order of graph traversal. | 
-| `durations`   | Returns the accumulated duration in seconds for each edge in order of graph traversal. | 
-| `costs`       | Returns the accumulated cost for each edge in order of graph traversal. | 
-| `edge_ids`   | Returns the internal edge IDs for each edge in order of graph traversal. Mostly interesting for debugging. | 
-| `statuses`   | Returns the edge states for each edge in order of graph traversal. Mostly interesting for debugging. Can be one of "r" (reached), "s" (settled), "c" (connected). |
+| `distance`   | Returns the accumulated distance in meters for each edge in order of graph traversal. | 
+| `duration`   | Returns the accumulated duration in seconds for each edge in order of graph traversal. | 
+| `cost`       | Returns the accumulated cost for each edge in order of graph traversal. | 
+| `edge_id`   | Returns the internal edge IDs for each edge in order of graph traversal. Mostly interesting for debugging. | 
+| `pred_edge_id` |  Returns the internal edge IDs of the predecessor for each edge in order of graph traversal. Mostly interesting for debugging. |
+| `edge_status`   | Returns the edge states for each edge in order of graph traversal. Mostly interesting for debugging. Can be one of "r" (reached), "s" (settled), "c" (connected). |
```

Comment:
one day we'll have to make this no longer beta and lock down the api :smile:

Reply:
I’d be fine with lifting the BETA status actually. I’ve used it now quite a bit. Apart from that one bug where random edges are included for isochrones, which I’d really like to fix first.